### DISCUSSION TITLE
When the debugger is running, have it provide input/feedback into IntelliSense

### DISCUSSION THREAD

**Discussion by greazer** [microsoft/pylance-release#5291] [Enhancement]
Highly related to microsoft/vscode-jupyter#1561, our new ability to debug into cells, makes this issue even more obvious than it was before. 

Repro:
1) src/test/datascience/manualTestFiles/manualTestFile.py
2) Execute to the cell beginning with: 
myNparray = np.array([['Bob', 1, 2, 3], ['Alice', 4, 5, 6], ['Gina', 7, 8, 9]])
myDataFrame = pd.DataFrame(myNparray, columns=['name', 'b', 'c', 'd'])
3) Debug into the cell to instantiate myDataFrame.
4) Enter a newline in the cell and type "myDataFrame." 

Expected Result:
- Column names "name", "b", "c" and "d" are listed in auto-complete.

Actual Result:
- They are not.

However they DO appear in the Debug Console input box. 

Also note that techincally this is not a data science specific problem. The same thing can happen when doing regular debugging. 

We've had several customers notice the related issue microsoft/vscode-jupyter#1561 which deals specifically with the interactive window. As more jupyter like features are added to our experience this will become more of a problem, therefore, I'd really like to see this addressed. :)

**Comment by greazer** [comment]
This is not specific to data science. This is about enabling column autocomplete when doing regular debugging.

**Comment by greazer** [comment]
This is not specific to data science. This is about enabling column autocomplete when doing regular debugging.

**Comment by luabud** [comment]
@savannahostrowski  JFYI
